xml/sip_user_agents.xml

```diff
@@ -618,4 +618,81 @@
     <param pos="1" name="os.version"/>
   </fingerprint>
 
+  <fingerprint pattern="^(TAU-\d[\w.IP]+)/([\d.]+) SN/(VI\d+)$">
+    <description>Eltex TAU model VoIP gateway - with serial number</description>
+    <example hw.product="TAU-8.IP" os.version="2.6.3">TAU-8.IP/2.6.3 SN/VI12345678</example>
+    <example os.version="2.0.0.229" hw.serial_number="VI12345678">TAU-4M.IP/2.0.0.229 SN/VI12345678</example>
+    <param pos="0" name="os.vendor" value="Eltex"/>
+    <param pos="0" name="os.product" value="{hw.product} Firmware"/>
+    <param pos="2" name="os.version"/>
+    <param pos="0" name="os.device" value="VoIP Gateway"/>
+    <param pos="0" name="hw.vendor" value="Eltex"/>
+    <param pos="1" name="hw.product"/>
+    <param pos="3" name="hw.serial_number"/>
+    <param pos="0" name="hw.device" value="VoIP Gateway"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(TAU-\d[\d.IP]+)/([\d.]+) SN/(VI\w+) (?:SHA/[0-9a-f]+ )?sofia-sip/([\d.]+)$">
+    <description>Eltex TAU model VoIP gateway - with serial number and sofia version</description>
+    <example hw.product="TAU-8.IP" hw.serial_number="VI12345678">TAU-8.IP/2.3.0 SN/VI12345678 sofia-sip/1.12.10</example>
```

Comment:
Verification of TAU firmware numbering scheme: https://eltex-co.com/catalog/tau-16-ip-en.php
Verification of `sofia-sip` version numbering scheme: https://github.com/freeswitch/sofia-sip/releases

Reply:
The character class will allow some combinations that do not appear to be valid products, for example, `TAU-8.I` or `TAU-8.P`. Should the `.IP` be outside of the character class?
```
(TAU-\d[\d.IP]+)
```

Reply:
An outlier from the one or two digits followed by ".IP" is the [TAU-32M.IP](https://eltex-co.com/catalog/tau-32m-ip-en.php).

Reply:
RE: the business routers: Their primary function is as a VoIP gateway and have VoIP gateway in the product description. 

RE: TAU-32M.IP - I don't have a SIP User Agent example for this product. There are quite a few banners yet to be added for this vendor so I'd expect this to be added in the future if it is observed. I'd recommend re-fingerprinting existing data or releasing a Recog version and pulling metrics from future Sonar study.

Reply:
I spoke with @tsellers-r7 and 
1. the business routers with VoIP support should be included
2. the character class concern is "it's low risk", but could regex could be updated
Tom doesn't have time to address these so I'll process the PR as-is and follow-up with another PR.